frontend/src/components/common/header/header.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import iconLogo from 'assets/images/icon-logo.svg';
+import { Button, Link } from '../index';
+import {
+  AppRoute,
+  ButtonColor,
+  ButtonIcon,
+  ButtonStyleType,
+  ButtonType,
+} from 'common/enums';
+import { Props, useVisible } from './common';
+import styles from './header.module.scss';
+
+const Header: React.FC<Props> = ({ user }) => {
+  const { ref, isVisible, setIsVisible } = useVisible(false);
+
+  const handleSearchClick = () => {
+    // TODO: add search logic
+  };
+
+  const handleNotificationsClick = () => {
+    // TODO: add notification logic
+  };
+
+  const toggleShowMenu = () => {
+    setIsVisible(!isVisible);
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.logoSection}>
+        <Link to={AppRoute.ROOT}>
+          <img src={iconLogo} loading="lazy" alt="HealthCare Logo" />
+          <div className={styles.logoLabel}>HealthCare</div>
+        </Link>
+      </div>
+      <div className={styles.pagesSection}>
+        <Link to={AppRoute.DOCTOR}>Doctor</Link>
+        <Link to={AppRoute.CLINIC}>Clinic</Link>
+        <Link to={AppRoute.MY_PROFILE}>My Profile</Link>
+        <Link to={AppRoute.MY_CALENDAR}>My Calendar</Link>
+        <Link to={AppRoute.MY_CHATS}>My Chats</Link>
+      </div>
+      <div className={styles.userSection}>
+        <Button
```

Comment:
Add props `href` to the `Button` component

Than in the `Button` component do something like

```
const isLink = Boolean(href);

return isLink ? <link /> : <button />
```